helpers/path.go

```diff
@@ -124,7 +127,7 @@ func (p *PathSpec) UnicodeSanitize(s string) string {
 	for i, r := range source {
 		if r == '%' && i+2 < len(source) && ishex(source[i+1]) && ishex(source[i+2]) {
 			target = append(target, r)
-		} else if unicode.IsLetter(r) || unicode.IsDigit(r) || unicode.IsMark(r) || r == '.' || r == '/' || r == '\\' || r == '_' || r == '-' || r == '#' || r == '+' || r == '~' {
+		} else if unicode.IsLetter(r) || unicode.IsDigit(r) || unicode.IsMark(r) || r == '.' || r == '/' || r == '\\' || r == '_' || r == '-' || r == '#' || r == '+' || r == '~' || r == ' ' {
```

Comment:
This should be superflous if you adjust the order of the above.

Reply:
Please excuse my ignorance, but I can't seem to reorder the operations in the `MakePath` function to make adding the space to the `UnicodeSanitize` unnecessary.

Changing to: Regex ReplaceAll -> UnicodeSanitize -> Trim 
causes the test case: `{"Foo & bar", "Foo-bar", true}` to fail.